src/prpy/base/wam.py

```diff
@@ -31,6 +31,7 @@
 import logging
 import numpy
 import openravepy
+import time
```

Comment:
True. We added the import as part of a change that was eventually removed. However, when I removed that change I noticed the ServoTo function uses the time module without importing it. So I kept the import. I have now moved it down into the ServoTo function where it is needed.